.github/workflows/shared-release-pull-request-java.yml

```diff
@@ -259,7 +269,7 @@ jobs:
     env:
       environmentName: staging01
       environmentTag: staging01
-      environmentUrl: http://app.test01/
+      environmentUrl: http://app.staging01/
```

Comment:
was this unused or just nobody noticed?

Reply:
I think it is only used for Slack notification links, and staging deploy notifications were disabled